### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,39 @@
 # Aldryn MailChimp
```
Comment (evildmp) [inline README.md]: README should be in RST by preference.

File: addon.json [hunk 2]
```diff
@@ -1,19 +1,8 @@
...
-    "author": {
-        "name": "Divio AG",
-        "url": "https://www.divio.ch"
-    },
-    "license": {
-        "name": "BSD",
-        "text": "!literal-include LICENSE.txt"
-    }
+    ]
 }
```
Comment (GaretJax) [inline addon.json]: Missing newline :)

**Comment by GaretJax** [comment]
LGTM, but @evildmp has some reservations :)

**Comment by intellectronica** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (intellectronica) [inline README.md]: That's our standard, but I don't think we should block on that, since this is already an improvement.

**Comment by mkoistinen** [comment]
Thanks, guys.

**Pull request merged**